File: 2018-2/ap1/exerciciosaula03/aula3ex19.c
#include <stdio.h>
#include <conio.h>

int main(){

int cod1, cod2, quan1, quan2;
float vl1, vl2, vtc1, vtc2, vt;

   printf("Qual eh o codigo da primeira peca? \n");
   scanf("%i", &cod1);
   scanf("%i", &cod2);

   printf("\nSao quantas pecas do produto %i? ",cod1);
   scanf("%i", &quan1);
   printf("\nSao quantas pecas do produto %i? ",cod2);
   scanf("%i", &quan2);

   printf("\nQual eh o valor do produto %i? ",cod1);
   scanf("%f", &vl1);
   printf("\nQual eh o valor do produto %i? ",cod2);
   scanf("%f", &vl2);

   vtc1 = quan1 * vl1;
   vtc2 = quan2 * vl2;
   vt = vtc1 + vtc2;

   printf("\nVoce comprou %i unidades do produtos codigo %i no valor de R$%.2f e %i unidades do produtos codigo %i no valor de R$%.2f.", quan1, cod1, vl1, quan2, cod2, vl2);

   printf("\nDando um total de R$%.2f.",vt);

}

File: 2018-2/ap1/exerciciosaula03/aula3ex19.py
cod1 = int(input('Qual é o código da primeira peça? '))
cod2 = int(input('Qual é o código da segunda peça? '))

quan1 = int(input('São quantas peças do produto {}? '.format(cod1)))
quan2 = int(input('São quantas peças do produto {}? '.format(cod2)))

vl1 = float(input('Qual é o valor do produto {}? '.format(cod1)))
vl2 = float(input('Qual é o valor do produto {}? '.format(cod2)))

vtc1 = quan1 * vl1
vtc2 = quan2 * vl2
vt = vtc1 + vtc2

print('Você comprou {} unidades do produtos codigo {} no valor de R$ {:.2f} e {} unidades do produtos codigo {} no valor de R$ {:.2f}\n'.format(quan1, cod1, vl1, quan2, cod2, vl2))
print('Dando um total de R$ {:.2f}.'.format(vt))
